### PR TITLE
Add Schlesi network with Lighthouse

### PR DESCRIPTION
**Pull request by morrigan**

### CODE REVIEW

File: src/renderer/actions/network.ts [hunk 1]
```diff
@@ -23,12 +24,15 @@ export const saveSelectedNetworkAction = (network: string): ISaveSelectedNetwork
...
     return async (): Promise<void> => {
         switch(network) {
             case SupportedNetworks.PRYSM:
                 await BeaconChain.startPrysmBeaconChain(ports);
                 break;
+            case SupportedNetworks.SCHLESI:
+                await BeaconChain.startSchlesiBeaconChain(ports);
+                break;
             default:
                 await BeaconChain.startPrysmBeaconChain(ports);
```
Comment (mpetrunic) [inline src/renderer/actions/network.ts:37]: Make schlesi default

File: src/renderer/actions/network.ts [hunk 2]
```diff
@@ -61,6 +65,7 @@ export interface ILoadedValidatorBeaconNodesAction {
 export const loadValidatorBeaconNodes = (validator: string, subscribe = false) => {
     return async (dispatch: Dispatch<Action<unknown>>, getState: () => IRootState): Promise<void> => {
         const validatorBeaconNodes = await getState().auth.account!.getValidatorBeaconNodes(validator);
+        console.log("validatorBeaconNodes: ", validatorBeaconNodes);
```
Comment (mpetrunic) [inline src/renderer/actions/network.ts:68]: use logger

File: src/renderer/containers/Onboard/Configure/ConfigureContainer.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ interface IInjectedProps {
 
 const ConfigureContainerComponent: React.FunctionComponent<IOwnProps & IInjectedProps> = (props) => {
     const [beaconNodeInput, setBeaconNodeInput] = useState("");
-    const [selectedNetworkIndex, setSelectedNetworkIndex] = useState(0);
+    const [selectedNetworkIndex, setSelectedNetworkIndex] = useState(1);
```
Comment (mpetrunic) [inline src/renderer/containers/Onboard/Configure/ConfigureContainer.tsx:22]: maybe use enum or something here?

File: src/renderer/services/docker/images.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import {SupportedNetworks} from "../eth2/supportedNetworks";
+import {BeaconChain} from "./chain";
+import {IDockerRunParams} from "./type";
+
+const eth1Endpoint = "https://goerli.infura.io/v3/9b8caef145c74574869579199c47e847";
+
+export const getLighthouseBeaconChainConfig = (network: SupportedNetworks): IDockerRunParams => ({
```
Comment (mpetrunic) [inline src/renderer/services/docker/images.ts:7]: That's not really beacon chain config but rather docker config. I would maybe integrate this in networks configuration, but split into different files

Comment (mpetrunic) [reply src/renderer/services/docker/images.ts:7]: Like:
```
/services/networks
   /index.ts with network cofniguration
   /schlesi.ts
   /local.ts
   etc
```

File: src/renderer/services/eth2/networks.ts [hunk 5]
```diff
@@ -28,6 +28,18 @@ const networks: INetworkConfig[] = [
...
+        networkName: SupportedNetworks.SCHLESI,
+        networkId: 5,
+        contract: {
+            address: "0xA15554BF93a052669B511ae29EA21f3581677ac5",
+            depositAmount: 32,
+            bytecode: "0x",
+            deployedAtBlock: 2596126
+        },
+        eth2Config: mainnetBeaconConfig,
+        eth1Provider: ethers.getDefaultProvider("goerli")
```
Comment (mpetrunic) [inline src/renderer/services/eth2/networks.ts:41]: We could have clients and docker config

**Comment by morrigan** [comment]
@mpetrunic this is ready now :) I guess we can merge after spec 11 ?

File: src/renderer/services/eth2/networks/prysm.ts [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+import {ethers} from "ethers";
+import {BeaconChain} from "../../docker/chain";
+import {IDockerRunParams} from "../../docker/type";
+import {SupportedNetworks} from "../supportedNetworks";
+import {INetworkConfig} from "../../interfaces";
+
+const dockerConfig: IDockerRunParams = {
+    name: BeaconChain.getContainerName(SupportedNetworks.PRYSM),
+    image: "gcr.io/prysmaticlabs/prysm/beacon-chain:latest",
+    restart: "unless-stopped",
```
Comment (mpetrunic) [inline src/renderer/services/eth2/networks/prysm.ts:11]: on-failure is maybe better?

File: src/renderer/services/eth2/networks/schlesi.ts [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+import {IDockerRunParams} from "../../docker/type";
+import {SupportedNetworks} from "../supportedNetworks";
+import {INetworkConfig} from "../../interfaces";
+
+const eth1Endpoint = "https://goerli.infura.io/v3/9b8caef145c74574869579199c47e847";
+
+const dockerConfig: IDockerRunParams = {
+    name: BeaconChain.getContainerName(SupportedNetworks.SCHLESI),
+    image: "sigp/lighthouse:latest",
+    restart: "unless-stopped",
```
Comment (mpetrunic) [inline src/renderer/services/eth2/networks/schlesi.ts:10]: on-failure is maybe better?

**Pull request merged**